Mathlib/Algebra/Group/Pointwise/Finset/Basic.lean

```diff
@@ -1628,6 +1648,25 @@ theorem card_le_card_mul_self' {s : Finset α} : s.card ≤ (s * s).card := by
 
 end
 
+section CancelMonoid
+variable [DecidableEq α] [CancelMonoid α] {s : Finset α} {m n : ℕ}
+
+@[to_additive]
+protected lemma Nonempty.card_pow_mono (hs : s.Nonempty) : Monotone fun n : ℕ ↦ (s ^ n).card :=
```

Comment:
gcongr on (a version of) this and below?

Reply:
I already tried and `gcongr` doesn't like it, probably because it's the composition of two functions:
```
@[gcongr] attribute only applies to lemmas proving f x₁ ... xₙ ∼ f x₁' ... xₙ'.
 Not all arguments are free variables in the conclusion of ∀ {α : Type u_2} [inst : DecidableEq α]
  [inst_1 : CancelMonoid α] {s : Finset α} {m n : ℕ}, m ≠ 0 → m ≤ n → #(s ^ m) ≤ #(s ^ n)
  ```

Reply:
Ah, okay... I wonder if this could be added to gcongr? @hrmacbeth, what do you think?